addons/product/security/ir.model.access.csv

```diff
@@ -31,7 +31,7 @@ access_product_label_layout_user,product.label.layout.user,model_product_label_l
 access_product_tag_manager,product.tag.manager,model_product_tag,group_product_manager,1,1,1,1
 access_product_combo_manager,product.combo.manager,model_product_combo,group_product_manager,1,1,1,1
 access_product_combo_item_manager,product.combo.item.manager,model_product_combo_item,group_product_manager,1,1,1,1
-access_product_document_user,access_product_document_user,model_product_document,base.group_user,1,1,1,1
+access_product_document_user,access_product_document_user,model_product_document,base.group_user,1,0,0,0

```

Comment:
I suppose you should give the rights to the product manager too?
(and to sale managers too, like it is given to mrp managers)